spec/api-browser-window-spec.ts

```diff
@@ -6533,8 +6530,7 @@ describe('BrowserWindow module', () => {
       );
     });
 
-    // FIXME(codebytere): figure out why these are failing on macOS arm64.
-    ifit(process.platform === 'darwin' && process.arch !== 'arm64')('Allows setting a transparent window via CSS', async () => {
+    ifit(hasCapturableScreen())('Allows setting a transparent window via CSS', async () => {
```

Comment:
This test is consistently failing on macOS arm64:
https://github.com/electron/electron/actions/runs/9943722345/job/27512304857?pr=42320#step:14:692

Reply:
Resolved in f31cb08